src/main/scala/cromwell/binding/RuntimeAttributes.scala

```diff
@@ -183,11 +209,25 @@ object RuntimeAttributes {
     }
   }
 
-  private def processRuntimeAttribute(ast: Ast): (String, String) = {
-    (ast.getAttribute("key").sourceString, ast.getAttribute("value").sourceString)
-  }
-
-  private def processRuntimeAttributes(astList: AstList): Map[String, String] = {
+  private def processRuntimeAttribute(ast: Ast): (String, Seq[String]) = {
+    val key = ast.getAttribute("key").sourceString
+    val seq = Option(ast.getAttribute("value")) map { valAttr =>
+      // TODO: NoFunctions should be converted to a case object or a constant
+      WdlExpression.evaluate(valAttr, NoLookup, new NoFunctions) match {
+        case scala.util.Success(wdlPrimitive: WdlPrimitive) => Seq(wdlPrimitive.valueString)
+        case scala.util.Success(wdlArray: WdlArray) => wdlArray.value.map(_.valueString)
+        case scala.util.Success(wdlValue: WdlValue) =>
```

Comment:
Because of `scalaz.Success` already used in `RuntimeAttributes.apply()`. I could alias the package if it would look better, like the builtin scala to java wrappers [do](https://github.com/scala/scala/blob/v2.11.7/src/library/scala/collection/convert/WrapAsScala.scala#L13)? To me that's a bit more cryptic.